src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -21,9 +21,13 @@ import { useSelectedSkillsAndJobSkills } from './data/hooks';
 import { useDefaultSearchFilters } from '../search/data/hooks';
 import { UserSubsidyContext } from '../enterprise-user-subsidy';
 
-const getCourseSkills = (course) => (
-  course.skill_names?.length > 0 ? course.skill_names.slice(0, MAX_VISIBLE_SKILLS_CARD) : null
-);
+const getCourseSkills = (course, selectedJobSkills) => {
+  let commonSkills = [];
+  if (course.skill_names?.length > 0) {
```

Comment:
Can we avoid this `if`  statement, and be more readable
```
course_skills = course.skills_names || []
let commonSkills = course_skills.filter(skill => selectedJobSkills.includes(skill))
return commonSkills.slice(0, MAX_VISIBLE_SKILLS_CARD);
```

Reply:
done